src/components/v2/TokenTextField/FormikTokenTextField.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useField } from 'formik';
+import { TokenTextField, ITokenTextFieldProps } from '.';
+
+interface IFormikTokenTextField extends Omit<ITokenTextFieldProps, 'name' | 'onChange' | 'value'> {
+  name: string;
+  errorCodes?: string[];
+}
+
+export const FormikTokenTextField = ({ name, errorCodes = [], ...rest }: IFormikTokenTextField) => {
+  const [{ value, onBlur }, { error }, { setValue }] = useField(name); // eslint-disable-line  @typescript-eslint/no-unused-vars
```

Comment:
It looks like we can get rid of the silencer comment, I ran the linter on your branch without it and it didn't return any error
```suggestion
  const [{ value, onBlur }, { error }, { setValue }] = useField(name);
```